test/modules/modal/components/Modal.test.js

```diff
@@ -30,49 +57,105 @@ describe('Modal', () => {
 		});
 	});
 
-	describe('when modal state changed', () => {
+	describe('when modal changes', () => {
 
-		it('adds content to modal from a string', async () => {
-			const element = await setup();
+		describe('modal.portrait', () => {
 
-			openModal('title', 'content');
+			it('adds the corresponding css class', async () => {
 
-			expect(store.getState().modal.data.title).toBe('title');
-			expect(store.getState().modal.data.content).toBe('content');
-			expect(element.shadowRoot.querySelector('.modal')).toBeTruthy();
-			expect(element.shadowRoot.querySelector('.modal__title').innerText).toBe('title');
-			//Note: Webkit appends a line break to the 'content' in this case
-			expect(element.shadowRoot.querySelector('.modal__content').innerText).toMatch(/content[\r\n]?/);
+				const state = {
+					media: {
+						portrait: false,
+						observeResponsiveParameter: true
+					}
+				};
+
+				const element = await setup(state);
+
+				openModal('title', 'content');
+
+				expect(element.shadowRoot.querySelector('.modal__container').classList).toContain('is-landscape');
+				expect(element.shadowRoot.querySelector('.modal__container').classList).not.toContain('is-portrait');
+
+				setIsPortrait(true);
+
+				expect(element.shadowRoot.querySelector('.modal__container').classList).not.toContain('is-landscape');
+				expect(element.shadowRoot.querySelector('.modal__container').classList).toContain('is-portrait');
+			});
 		});
 
-		it('adds content to modal from a lit-html TemplateResult', async () => {
-			const element = await setup();
+		describe('modal.data', () => {
+
+			it('adds content from a string', async () => {
 
-			const template = (str) => html`${str}`;
+				const state = {
+					media: {
+						portrait: false
+					}
+				};
 
-			openModal('title', template('content'));
+				const element = await setup(state);
 
-			expect(store.getState().modal.data.title).toBe('title');
-			expect(store.getState().modal.data.content).toBeInstanceOf(TemplateResult);
-			expect(element.shadowRoot.querySelector('.modal')).toBeTruthy();
-			expect(element.shadowRoot.querySelector('.modal__title').innerText).toBe('title');
-			//Note: Webkit appends a line break to the 'content' in this case
-			expect(element.shadowRoot.querySelector('.modal__content').innerText).toMatch(/content[\r\n]?/);
+				openModal('title', 'content');
+
+				expect(store.getState().modal.data.title).toBe('title');
+				expect(store.getState().modal.data.content).toBe('content');
+				expect(element.shadowRoot.querySelector('.modal')).toBeTruthy();
+				expect(element.shadowRoot.querySelector('.modal__title').innerText).toMatch(/title[\r\n]?/);
+				//Note: Webkit appends a line break to the 'content' in this case
+				expect(element.shadowRoot.querySelector('.modal__content').innerText).toMatch(/content[\r\n]?/);
```

Comment:
a updated 'Note'-comment  would be nice...i suggest to mention both expectations or use a more generalized form, to stay in touch with the changes